verticaltabs.js

```diff
@@ -126,6 +127,17 @@ VerticalTabs.prototype = {
         set('extensions.tabcentertest1@mozilla.com.lastUsedTimestamp', Date.now().toString());
         ss.setWindowValue(window, 'TCtoggledon', mainWindow.getAttribute('toggledon'));
         this.init();
+        if (mainWindow.getAttribute('F11-fullscreen') === 'true') {
+          let fullscreenctls = document.getElementById('window-controls');
+          let navbar = document.getElementById('nav-bar');
+          let toggler = document.getElementById('fullscr-toggler');
+          let sibling = document.getElementById('navigator-toolbox').nextSibling;
+          toggler.removeAttribute('hidden');
+          //hidden nav toolbox needs to be moved 1 pix higher to account for the toggler every time it hides
```

Comment:
We should remove this comment, cause we aren't adding or removing a pixel here anymore…